src/pytorch_lightning/trainer/trainer.py

```diff
@@ -2207,11 +2206,11 @@ def model(self, model: torch.nn.Module) -> None:
 
     @property
     def log_dir(self) -> Optional[str]:
-        if len(self.loggers) == 1:
-            if isinstance(self.logger, TensorBoardLogger):
-                dirpath = self.logger.log_dir
+        if len(self.loggers) > 0:
+            if isinstance(self.loggers[0], TensorBoardLogger):
+                dirpath = self.loggers[0].log_dir
```

Comment:
so the profiler output path and checkpoint path will not be the same when using `Tensorboard`.

Reply:
Yes, but this is a pre-existing issue that will be addressed in #14188.

Reply:
Yes, perfect!